tests/docs/smoke-all/2023/04/04/issue-5084.qmd

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
@cderv Is this testing the reveal code path as well?

Reply:
No only html format. We would need to add `revealjs` at least in `_quarto.yml` 

I'll add it. thanks

Reply:
Done in #5107